x/pylons/types/payment_processor_test.go

```diff
@@ -93,6 +114,27 @@ func TestPaymentProcessor_ValidatePaymentInfo(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "InvalidName",
+			fields: fields{
+				CoinDenom:            "ustripeusd",
+				PubKey:               base64.StdEncoding.EncodeToString(privKey.PubKey().Bytes()),
+				ProcessorPercentange: DefaultProcessorPercentage,
```

Comment:
ProcessorPercentage is correct spelling.

Reply:
Thank you very much! I have fix in commit https://github.com/Pylons-tech/pylons/pull/870/commits/8aa682a4948fce1f64f88c04f253cbb7e3b4e2f1